sevntu-checks/src/test/resources/com/github/sevntu/checkstyle/checks/coding/FieldDeclarationsCountCheckNOK.java

```diff
@@ -0,0 +1,12 @@
+package com.github.sevntu.checkstyle.checks.coding;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class FieldDeclarationsCountCheckNOK {
```

Comment:
NOK, because my understanding is that Checkstyle cannot handle this, because its based on analysing the AST of a single file; AFAIK I couldn't check parent class fields, right? So.. no point, I would have thought.